src/index.js

```diff
@@ -1,67 +1,152 @@
+    const checkbox = document.getElementById(`${element.index}-checkbox`);
+    checkbox.addEventListener('change', () => {
+      if (checkbox.checked) {
+        element.completed = true;
+        window.localStorage.setItem('tasklist', JSON.stringify(arr));
+      } else {
+        element.completed = false;
+        window.localStorage.setItem('tasklist', JSON.stringify(arr));
+      }
+    });
+  });
+}
 
-function out() {
-  let out = '';
-  for (const key in todoList) {
-    if (todoList[key].complited === true) {
-      out += `<input type='checkbox' checked `;
+function prepopstatus(arr) {
+  arr.forEach((element) => {
+    const checkbox = document.getElementById(`${element.index}-checkbox`);
+    if (element.completed === true) {
+      checkbox.checked = true;
     } else {
-      out += `<input type='checkbox'>`;
+      checkbox.checked = false;
     }
+  });
+}
 
-    out += todoList[key].description;
-    out += `<p id='ppp'></p><br>`;
+const form = document.getElementById('form');
+const taskinput = document.querySelector('.taskadder');
+const sync = document.querySelector('.sync');
+const entericon = document.querySelector('.enter-icon');
+const deletecompleted = document.getElementById('delcompleted');
+const list = [];
+let displayedList;
+
+const todoList = (arr) => {
+  arr.forEach((element) => {
+    const duties = document.getElementById('duties');
+    // Create task li //
+    duties
+      .appendChild(document.createElement('li'))
+      .setAttribute('id', element.index);
+    const task = document.getElementById(element.index);
+    task.classList.add('task', 'draggable');
+    // Create checkbox //
+    task
+      .appendChild(document.createElement('input'))
+      .setAttribute('id', `${element.index}-checkbox`);
+    const checkbox = document.getElementById(`${element.index}-checkbox`);
+    checkbox.setAttribute('type', 'checkbox');
+    checkbox.classList.add('checkbox');
+    // Create description //
+    task
+      .appendChild(document.createElement('p'))
+      .setAttribute('id', `${element.index}-description`);
+    const description = document.getElementById(`${element.index}-description`);
+    description.classList.add('description');
+    description.innerText = element.description;
+    // Create DragBtn //
+    task
+      .appendChild(document.createElement('i'))
+      .setAttribute('id', `${element.index}-drag`);
+    const dragBtn = document.getElementById(`${element.index}-drag`);
+    dragBtn.classList.add('fas', 'fa-ellipsis-v', 'drag-btn');
+    // create trashcan //
+    task
+      .appendChild(document.createElement('i'))
+      .setAttribute('id', `${element.index}-trash`);
+    const trashBtn = document.getElementById(`${element.index}-trash`);
+    trashBtn.classList.add('far', 'fa-trash-alt', 'trash-btn');
+    // Create add event listeners //
+    mousedown(dragBtn);
+  });
+  dragndrop(arr);
+  status(arr);
+  prepopstatus(arr);
+  edit(arr);
+  removetask(arr);
+};
+
```

Comment:
- [ ] I can't able to see any function which `add`, `edit`, or `delete` the task, Please make sure to add all the functionality in this PR and also I see you have added drag and drop functionality in this. Please make sure to work in the particular milestone.